pkg/core/alias.go

```diff
@@ -15,6 +18,7 @@ var (
 	NewLogger = kuma_log.NewLogger
 	SetLogger = kube_log.SetLogger
 	Now       = time.Now
+	LookupIP  = dns.MakeCaching(net.LookupIP, 10*time.Second)
```

Comment:
I'd like to avoid having it in `core`. This list of aliases should be used only for essential-critical-core stuff.
My suggestion, place it in the file with cache

Also by doing this I think you just dragged DNS server dependency on anyone who is importing `core` even just for logging (kuma-dp)